bioimageio/core/_resource_tests.py

```diff
@@ -1,8 +1,9 @@
 import traceback
 import warnings
 from itertools import product
-from typing import Dict, Hashable, List, Literal, Optional, Set, Tuple, Union
+from typing import Dict, Hashable, List, Literal, Optional, Set, Tuple, Union, TypedDict
 
+from attr import dataclass
```

Comment:
seems to be unused?

Reply:
Oops I think this was an automatically added import from my IDE before I switched to `TypedDict` for `PresicionArgs`, I will remove it